### DISCUSSION TITLE
Internal Server Error

### DISCUSSION THREAD

**Discussion by kevman289** [evroon/bracket#1042] [Q&A]
i cannot seem to log in. every time i try to, i get the same error and the backend container crashes

![Screenshot 2024-11-21 205824](https://github.com/user-attachments/assets/ab32e2f9-73ac-4e98-abff-853e97cf6579)

**Comment by evroon** [comment]
Could you provide logs of the backend and frontend containers and a browser network tab screenshot?

**Reply by 1337Rafael1337** [reply]
![networktab](https://github.com/user-attachments/assets/c5f678ad-2642-4774-9f25-9ea4fd5737c1)
![logs](https://github.com/user-attachments/assets/39902773-d85d-416d-babc-9e044c7d67c7)

**Reply by ghost** [reply]
Are you using any kind of reverse proxy (like NGINX) in front of your Bracket server, or are you accessing it via an IP:PORT in your browser?

**Reply by ypkoba** [reply]
<img width="1878" height="184" alt="Screenshot_2026-06-07_23-07-19" src="https://github.com/user-attachments/assets/d03c90ae-8b1d-4f50-8401-3ba9dd7ae2d0" />
I have the same issue. the only way I solved it was running it on the pi where the docker container was running. 
The browser wants to go to the localhost:8400 every time no matter what I set in the docker-compose file.
